pkgs/development/mobile/androidenv/tools.nix

```diff
@@ -6,7 +6,7 @@ deployAndroidPackage {
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optionals (os == "linux") [ autoPatchelfHook ];
   buildInputs = lib.optional (os == "linux") (
-      (with pkgs; [ glibc freetype fontconfig fontconfig.lib])
+      (with pkgs; [ glibc freetype fontconfig fontconfig.lib stdenv.cc.cc ])
```

Comment:
Is there a shorter reproducing expression than "[Try to rebuild with my system configuration](https://github.com/NixOS/nixpkgs/issues/226357)"?

I *think* you should be able to use `stdenv.cc.cc.libgcc or null` here, but I can't test it because I don't quite know how to reproduce the bug (without rebuilding all of @NotAShelf's NixOS install).

Reply:
I would love to be able to give you a shorter reproduction hint, but the only way I've been able to produce this is by building my system configuration with `diffoscope` in system packages.

Reply:
I didn't build the `diffoscope` to reproduce it. I just ran the tests. Take a look at the [README.md](https://github.com/NixOS/nixpkgs/blob/e8121fa7db800faac2e7160b1958d7125c030073/pkgs/development/mobile/androidenv/README.md) file.

Reply:
`libgcc` also works. check out https://github.com/NixOS/nixpkgs/pull/226489/commits/0ed63be46aacdb79f19b003e25ec08a2e0ef5a61